README.rst

```diff
@@ -21,19 +21,35 @@ Installation
 Obtaining via Conda
 -------------------
 
-The preferred way of obtaining DFTB+ is to install it via the conda package
+The preferred way of obtaining DFTB+ is to install it via the Conda package
```

Comment:
Their organization uses lower case for conda. https://docs.conda.io/projects/conda/en/latest/user-guide/index.html
```suggestion
The preferred way of obtaining DFTB+ is to install it via the conda package
```